doc/Authoring_Extensions.md

```diff
@@ -79,9 +77,9 @@ We also got a `lerna.json` file to configure `lerna`:
 
 ## Implementing the Extension
 
-Next let's look at the generated code for our extension in the `hello-world-extension` folder. Let’s start with the `package.json`. It specifies the package’s metadata, its dependencies to the (bleeding edge) theia core package, a few scripts and dev dependencies, and the theia-extensions.
+Next let's look at the generated code for our extension in the `hello-world-extension` folder. Let’s start with the `package.json`. It specifies the package’s metadata, its dependencies to the (bleeding edge) Theia core package, a few scripts and dev dependencies, and the theia-extensions.
 
-The keyword `theia-extension` is important: It allows a Theia app to identify and install Theia extensions from `npm`.
+The keyword `theiaExtensions` is important: It allows a Theia app to identify and install Theia extensions from `npm`.
```

Comment:
it was correct, the keyword is `theia-extension`, e.g.: https://github.com/theia-ide/theia/blob/55e431bd0ebff81a64a6ef1e53edc74dfb815bf6/packages/process/package.json#L18-L20

Reply:
@akosyakov reverted